Readme.md

```diff
@@ -161,3 +161,6 @@ script:
 
 ```
 
+## Known Issue
+
+Since this makes all seervices public, as a consequence errors like accessing private services in your code will not be caught in the tests.
```

Comment:
`s/seervices/services`

Reply:
Maybe add it more as a warning. "Be aware that if you make all services public then you will not get any errors if your bundles access public services".. or something like that.